test/sambah/local_utilities.py

```diff
@@ -0,0 +1,43 @@
+""" A module containing utility functionality used by the Harmony OPeNDAP
+    SubSetter regression tests. These functions are kept out of the Jupyter
```

Comment:
probably switch to SAMBA. The actual comparisons are different enough though, that I won't suggest merging them and adding them to the shared dir.